ocrd/ocrd/resolver.py

```diff
@@ -30,62 +34,77 @@ def download_to_directory(self, directory, url, basename=None, overwrite=False,
             directory (string): Directory to download files to
             basename (string, None): basename part of the filename on disk.
             url (string): URL to download from
-            overwrite (boolean): Whether to overwrite existing files with that name
+            if_exists (string, "skip"): Waht to do if target file already exists. One of ``skip`` (default), ``overwrite`` or ``raise``
```

Comment:
Typo *Waht*

Reply:
I thought those could be fixed by @stweil's next run with codespell ;-) Just joking, corrected, thanks.